src/azure-cli/azure/cli/command_modules/network/private_link_resource_and_endpoint_connections/resource_providers/batch_provider.py

```diff
@@ -0,0 +1,71 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from azure.cli.core.commands.client_factory import get_mgmt_service_client
+from azure.mgmt.batch import BatchManagementClient
+from azure.mgmt.batch.models import PrivateLinkServiceConnectionState, PrivateLinkServiceConnectionStatus
+from knack.log import get_logger
+from . import PrivateEndpointClient
+
+logger = get_logger(__name__)
+
+
+class BatchPrivateEndpointClient(PrivateEndpointClient):
```

Comment:
can you inherit from `class BatchPrivateEndpointClient(GeneralPrivateEndpointClient):` and just re-implement `remove_private_endpoint_connection` function?

Reply:
Yea. We will still need the custom update method though as well as ours is a PATCH instead of a PUT